src/plugins/auto/auto_executable_network.cpp

```diff
@@ -36,6 +36,29 @@ IE::Parameter AutoExecutableNetwork::GetConfig(const std::string& name) const {
     IE_THROW(NotFound) << name << " not found in the ExecutableNetwork config";
 }
 
+IE::Parameter AutoExecutableNetwork::GetMetric(const std::string& name, const std::string& target_device) const {
+    if (target_device == _autoSchedule->_loadContext[ACTUALDEVICE].deviceInfo.deviceName) {
+        if (_autoSchedule->_loadContext[ACTUALDEVICE].future.valid()) {
+            _autoSchedule->_loadContext[ACTUALDEVICE].future.wait();
+        }
+        if (_autoSchedule->_loadContext[ACTUALDEVICE].isAlready) {
+            try {
+                return _autoSchedule->_loadContext[ACTUALDEVICE].executableNetwork->GetMetric(name);
+            } catch (...) {
```

Comment:
why ... ? I suppose we explicitly know type of thrown exception

Reply:
Updated.